casanovo/config.yaml

```diff
@@ -1,30 +1,50 @@
 ###
 # Casanovo configuration.
-# Blank entries are interpreted as "None"
+# Blank entries are interpreted as "None".
+# Parameters that can be modified when running inference with Casanovo,
+# i.e. denovo and eval modes in CLI, are marked with "(I)". Other parameters
+# shouldn't be changed unless a new Casanovo model is being trained.
 ###
 
 # Random seed to ensure reproducible results.
 random_seed: 454
 
 # Spectrum processing options.
+# Max number of peaks in spectrum
 n_peaks: 150
+# Min peak m/z allowed
```

Comment:
Clarify that spectra with too few peaks are skipped.

Reply:
We don't have a minimum number of peaks variable, spectra are skipped only if no peaks are left after pre-processing.